.github/workflows/test.yml

```diff
@@ -17,14 +17,19 @@ concurrency:
 jobs:
   pytester:
     runs-on: ${{ matrix.os }}
-    environment:
-      name: test
     strategy:
       fail-fast: false
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
         os: [ubuntu-latest, macOS-latest, windows-latest]
         backend: ["local", "db"]
+    env:
+      CACHIER_TEST_HOST: "localhost"
+      CACHIER_TEST_PORT: "27017"
+      #CACHIER_TEST_DB: "dummy_db"
+      #CACHIER_TEST_USERNAME: "myuser"
+      #CACHIER_TEST_PASSWORD: "yourpassword"
+      CACHIER_TEST_VS_LIVE_MONGO: "true"
```

Comment:
not sure what this one shall do? @shaypal5

Reply:
I'm not sure myself. Maybe change it to "CACHIER_TEST_VS_DOCKERIZED_MONGO". And we can bring back this one if we ever want to test against a live MongoDB again.